src/imap/mod.rs

```diff
@@ -229,19 +229,7 @@ impl Imap {
                 }
             }
             Err(err) => {
-                let message = {
-                    let config = &self.config;
-                    let imap_server: &str = config.lp.server.as_ref();
-                    let imap_port = config.lp.port;
-                    context
-                        .stock_string_repl_str2(
-                            StockMessage::ServerResponse,
-                            format!("IMAP {}:{}", imap_server, imap_port),
-                            err.to_string(),
-                        )
-                        .await
-                };
-                bail!("{}: {}", message, err);
+                bail!(err);
```

Comment:
ServerResponse is never shown to the user anymore now. It did not fit, the same, lengthy error message would have been shown multiple times.

Maybe we should just get rid of it, I think that I was not the only one who did not like it.
(it was the one with "Some providers email you additional info ... consult your provider or friends).